plugins/kotlin/uast/uast-kotlin-base/test/org/jetbrains/uast/test/common/kotlin/UastResolveApiFixtureTestBase.kt

```diff
@@ -64,25 +58,29 @@ class KotlinUastResolveApiTest : KotlinLightCodeInsightFixtureTestCase() {
 
         val resolvedDeclaration = functionCall.multiResolve()
         val resolvedDeclarationsStrings = resolvedDeclaration.map { it.element.text ?: "<null>" }
-        assertContainsElements(
-            resolvedDeclarationsStrings,
-            "fun foo(): Int = TODO()",
-            "fun foo(a: Int): Int = TODO()",
-            "fun foo(a: Int, b: Int): Int = TODO()"
-        )
+        // TODO: multiResolve
+        if (!isFirUastPlugin) {
```

Comment:
I do not like this approach with a lot of `if (!isFirUastPlugin) { assertSmth() }` in  tests. It does not seem to be very flexible and does not allow to easily find out that some functionality started to work.

What do you think about muting non-passing tests on the `FirUastResolveApiFixtureTest.test*` side? Something like 
```
 fun testMultiResolve() {
   considerFialing {
     checkMultiResolve(myFixture)
  }
}
```
Where `considerFailing` also checks if test started to pass. 

Actually I'm for the generated tests based on a testdata as they are more well structured and can handle such cases well. But considering that there are already a lot of non-generated tests, that approach should work too.

Reply:
I also noticed that this won't raise a flag if things start to work, thanks for pointing that out.

Added one more commit that removes plugin-specific assertions and rather adds a logic to check if it is expected to fail or notify users to fix whitelist.